epoxy-adapter/build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
   implementation rootProject.deps.androidAnnotations
   implementation rootProject.deps.androidRecyclerView
   implementation rootProject.deps.androidDesignLibrary
+  implementation rootProject.deps.kotlin
```

Comment:
Like Checkstyle for Java, do you want to bring ktlint for the Kotlin code ?
However here will be a bunch of code to re-format it the annotation processor that is already in Kotlin.

Reply:
I think ktlint would be great - we would just have to reformat all existing kotlin code to match.

I don't feel like doing it in this PR (since there is already a lot of other kotlin code), but it would be a good thing to do soon.

Thanks for bringing it up, I'll create an issue.